src/main/scala/GithubRelease.scala

```diff
@@ -47,33 +57,52 @@ case object GithubRelease {
       typeMap.getContentType
     }
 
-    def ghreleaseGetCredentials: DefTask[GitHub] = Def.task {
+    private def readCredentialsFrom(file: File): Option[GitHubCredentials] = {
+      val credentialsFile = Option(file)
+        .filter(_.isFile)
+        .filter(_.canRead)
+
+      val maybeCredentialParameters = credentialsFile.map { githubCredentialsFile =>
+        val lines = IO.readLines(githubCredentialsFile)
+        lines map { line =>
+          line.split("=")
+        } filter {
+          _.length == 2
+        } map { lineElements =>
+          lineElements(0) -> lineElements(1)
+        } toMap
+      }
```

Comment:
Btw, you could replace it with something like

```scala
import scala.collection.JavaConverters._

val props = new java.util.Properties()
props.load(new java.io.FileInputStream(githubCredentialsFile))
props.asScala //: Map[String, String]
```

Reply:
ah, so the format of the `.github` file matches that of a properties file? That simplifies things quite a bit 👍